release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -1635,15 +1635,60 @@ can not access the subscription resource. (link:https://bugzilla.redhat.com/show
 To upgrade an existing {product-title} 4.3 cluster to this latest release, see
 xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
 
+[[ocp-4-3-3]]
+=== RHBA-2020:0528 - {product-title} 4.3.3 Bug Fix Update
+
+Issued: 2020-02-24
+
+{product-title} release 4.3.3 is now available. The list of packages included in
+the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:0527[RHBA-2020:0527] advisory.
+The container images and bug fixes included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:0528[RHBA-2020:0528] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/4853201[{product-title} 4.3.3 container image list]
+
+[[ocp-4-3-3-bug-fixes]]
+==== Bug Fixes
+
+* The API group of KnativeServing resources `serving.knative.dev` is deprecated
+and it has changed to `operator.knative.dev` in Serverless Operator 1.4.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1779469[*BZ#1779469*])
+
+[[ocp-4-3-3-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.3 cluster to this latest release, see
+xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[[RHSA-2020-0562]]
+=== RHsA-2020:0562 - Moderate: OpenShift Container Platform 4.3 Security Update
+
+Issued: 2020-02-24
+
+An update for jenkins-slave-base-rhel7-container is now available for
+{product-title} 4.3. Details of the update are documented in the
+link:https://access.redhat.com/errata/RHSA-2020:0562[RHSA-2020:0562] advisory.
+
 ////
 // TODO: Cody/Andrew - please add the following note after the "Upgrading" section for 4.3.3, and keep the note for all 4.3.z releases going forward.
 
 [IMPORTANT]
 ====
-If you are upgrading to this release from {product-title} 4.2 or {product-title} 4.3.2 or earlier, you must restart all Pods after the upgrade is complete.
+If you are upgrading to this release from {product-title} 4.2 or {product-title}
+4.3.4 or earlier, you must restart all Pods after the upgrade is complete.
```

Comment:
@sheriff-rh Can you change this from 4.3.4 to 4.3.3? Thanks!

Reply:
Oh whoops, I got carried away. Good catch @bergerhoffer ! Fixing now.